linux_os/guide/services/sssd/sssd_enable_pam_services/ansible/shared.yml

```diff
@@ -0,0 +1,41 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = medium
+
+- name: Find all the conf files inside the /etc/sssd/conf.d/ directory
```

Comment:
Could you include the rule_title prefix in task names, please?
https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#ansible